book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -161,7 +161,7 @@ index.html |    1 +
 
 Das sieht ein bisschen anders aus, als das Merging mit dem `hotfix`-Branch, das Sie zuvor gemacht haben.
 Hier hat sich der Entwicklungsverlauf an einem früheren Zustand geteilt.
-Da der Commit auf dem Branch, auf dem Sie sich gerade befinden, kein unmittelbarer Vorgänger der Branch ist, in den Sie fusionieren, muss Git einige Arbeiten erledigen.
+Da der Commit auf dem Branch, auf dem Sie sich gerade befinden, kein unmittelbarer Vorgänger des Branches ist, in den Sie fusionieren, muss Git einige Arbeiten erledigen.
```

Comment:
```suggestion
Da der Commit auf dem Branch, auf dem Sie sich gerade befinden, kein unmittelbarer Vorgänger des Branchs ist, in den Sie fusionieren, muss Git einige Arbeiten erledigen.
```

Reply:
I should have said `branches`now, but I'm sure you have more experience

Reply:
Zuerst einmal Danke für deinen Beitrag.

Da wir hier sprachspezifische "Probleme" diskutieren, ist es einfacher und genauer das in der Muttersprache zu formulieren. Deshalb antworte ich hier auf Deutsch.

zu "Branches": das wäre der Plural

Gedacht als Hilfe, nicht als "Vorschrift" ist die Wortliste am Ende der Datei [TRANSLATION_NOTES_DE](https://github.com/progit/progit2-de/blob/master/TRANSLATION_NOTES_DE.asc). Sie kann jederzeit angepasst und erweitert werden.